app/Application.php

```diff
@@ -9,36 +9,42 @@
 
 class Application
 {
+    protected $messages;
     protected $historyRepository;
 
     public function __construct()
     {
+        $DS = DIRECTORY_SEPARATOR;
+
+        $this->messages = file_get_contents("locale{$DS}en.json");
+        $this->messages = json_decode($this->messages, true);
```

Comment:
let's move this logic to the separate `loadLocale` method

```suggestion
        $this->messages = $this->laodLocale();
```